src/api/util.rs

```diff
@@ -0,0 +1,20 @@
+use actix_web::web;
+use mongodb::{bson::Document, Client, Collection};
+
+// TODO: add check for valid catalogs
+// retrieves a mongodb collection
+pub fn get_collection(
```

Comment:
as mentioned in a previous comment, perhaps this could just take the db as parameter and the collection name. But quite frankly once we just do `db.collection(...)` we just don't need that function anymore